src/index.html

```diff
@@ -11,7 +11,168 @@
+        for="surname">
+          "Surname:"
+          <input 
+            type="text" 
+            autocomplete="off" 
+            name="surname" 
+            minlength="2" 
+            maxlength="30" 
+            required
+          >
+        </label>
+        <br>
+        <label class="form-label" for="name">
+          "Name:"
+          <input 
+            type="text" 
+            autocomplete="off" 
+            name="name" 
+            minlength="2" 
+            maxlength="30" 
+            required
+          >
+        </label>
+        <br>
+        <label class="form-label" for="age">
+          "How old are you? "
+          <input 
+            type="number" 
+            value="12" 
+            min="1" 
+            max="100" 
+            name="age" 
+            required
+          >
+        </label>
+        <br>
+        <label class="form-label" for="dob">
+          "Full data of birth"
+          <input 
+            type="date" 
+            name="dob" 
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="box-container">
+        <legend>Registration</legend>
+        <label class="form-label" for="email">
+          "Email:"
+          <input 
+            type="email" 
+            placeholder="email@example.com" 
+            name="email" 
+            required
+          >
+        </label>
+        <br>
+        <label class="form-label" for="password">
+          "Password"
+          <input  
+            type="password" 
+            name="password" 
+            minlength="8" 
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="box-container">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-label" for="fact">
+          "Do you love cats? "
+          <label class="form-label" for="yes">
+            <input
+              type="radio" 
+              name="yes" 
+              value="yes"
+            >
+            "Yes "
+          </label>
+          <label class="form-label" for="no">
+            <input 
+              type="radio" 
+              name="no" 
+              value="no"
+            >
+            "No"
+          </label>
+        </label>
+        <br>
+        <label class="form-label" for="color">
+          "What is your favorite color?"
+          <input 
+            type="color"
+          >
+        </label>
+        <br>
+        <label class="form-label" for="time">
+          "What time do you got to bed? "
+          <input 
+            type="time"
+          >
+        </label>
+        <br>
+        <label class="form-label" for="cars">
+          "What are your favorite brand of cars? "
+          <select 
+            name="cars" 
+            id="cars" 
+            multiple 
+          >
+            <option value="BMW">BMW</option>
+            <option value="AUDI">AUDI</option>
+            <option value="LADA">LADA</option>
```

Comment:
better use lower-case values in attribute `value` - that is a common approach to work with data on back-end:
```suggestion
            <option value="bmw">BMW</option>
            <option value="audi">AUDI</option>
            <option value="lada">LADA</option>
```